<launch>
    <arg name="namespace" default="yk_creator"/>
    <arg name="num_trials" default="10"/>
    <arg name="verbose" default="true"/>
    <group ns="$(arg namespace)">
        <node pkg="vibro_tactile_toolbox" name="collect_tactile_data_new" type="collect_tactile_data_new.py" output="screen">
            <param name="namespace" value="$(arg namespace)"/>
            <param name="root_pwd" value="$(find vibro_tactile_toolbox)"/>
            <param name="num_trials" value="$(arg num_trials)"/>
            <param name="verbose" value="$(arg verbose)"/>
        </node>
    </group>
</launch>